src/foremast/consts.py

```diff
@@ -155,7 +156,8 @@ def find_config():
                                  default='').split(','))
 GITLAB_TOKEN = validate_key_values(config, 'credentials', 'gitlab_token')
 SLACK_TOKEN = validate_key_values(config, 'credentials', 'slack_token')
-
+DEFAULT_TASK_TIMEOUT = validate_key_values(config, 'task_timeouts', 'default', default=120)
+TASK_TIMEOUTS = json.loads(validate_key_values(config, 'task_timeouts', 'envs', default="{}"))
```

Comment:
the intent is to allow timeouts to differ per environment and per task.  ConfigParser requires that you convert any non-string values to the correct type.  eval'ing a dictionary seemed worse than loading json.  

the json.loads call is also why the {} is quoted:

```>>> json.loads({})
TypeError: the JSON object must be str, bytes or bytearray, not 'dict'
```

We could instead represent timeouts as a section per environment, something like:

```
[dev_task_timeouts]
Task1=240
Task2=480

[stage_task_timeouts]
Task1=500```